src/components/calendar/Calendar.js

```diff
@@ -116,254 +190,122 @@ class Calendar extends React.Component {
-    let calendarApi = this.calendarComponentRef.current.getApi();
-    // Displays a datetime picker to set a future meeting
-    const myInput = document.querySelector("#futureButton");
-    // Had to put the confirmation and state update in the onClose callback as execution continues even after the flatpickr is displayed
-    const fp = flatpickr(myInput, {
-      position: "below",
-      enableTime: true,
-      noCalendar: false,
-      dateFormat: "H:i",
-      timeZone: "local",
-      onClose: () => {
-        // If the user canceled the picker the dates will be empty and there is nothing to do
-        if (fp.selectedDates.length === 0) {
-          return;
-        }
-        swal({
-          title: "Schedule Meeting?",
-          text: "Meeting will be added to the calendar!",
-          icon: "warning",
-          buttons: true,
-          dangerMode: false
-        }).then(scheduleAppointment => {
-          if (scheduleAppointment) {
-            // If the user confirmed the scheduled meeting we navigate to the date and update state
-            calendarApi.gotoDate(fp.selectedDates[0]);
+            });
             this.setState({
               ...this.state,
-              events: [
-                ...this.state.events,
-                { title: "Meeting", start: fp.selectedDates[0] }
-              ]
+              events: newEvents
             });
-            swal("Meeting has been added to the calendar!", {
+            swal(`Meeting date has been changed to ${info.event.start}`, {
               icon: "success"
             });
           } else {
-            swal("Cancelled, your meeting has not been set!");
+            //* if cancel is clicked, revert change in calendar API to previous date
+            info.revert();
+            swal("Cancelled, your meeting has not been changed!");
           }
-        });
-      }
+        })
+        .catch(err => console.log(err));
+    } catch (err) {
+      console.log(err);
+    }
+  };
+
+  handleEventClick = async info => {
+    console.log("info", info);
+    const meetingRef = firestore.collection("meetings").doc(info.event.id);
+    const meeting = meetingRef.get().then(doc => {
+      const start = new Date(doc.data().start.seconds * 1000);
+      this.setState({
+        ...this.state,
+        clickedMeeting: {
+          ...doc.data(),
+          start: start
+        }
+      });
     });
-    fp.open();
+    this.toggleModal();
   };
 
   handleDateClick = async arg => {
+    console.log("arg.date", arg.date);
     // Display only the time component of flatpickr so the user can select the meeting start time.  Like above the work is done in the
     // onClose function of flatickr as execution does not halt after the time picker is opened to allow the user to select a date.
-    console.log("handleDateClick triggered");
     let meetingDate = await new Date(
       arg.date.getFullYear(),
       arg.date.getMonth(),
       arg.date.getDate(),
-      0,
+      12,
```

Comment:
Updated so when an empty calendar day is clicked the default time is noon instead of midnight.